src/main/java/com/coveros/selenified/selenium/element/Element.java

```diff
@@ -0,0 +1,870 @@
+/*
+ * Copyright 2017 Coveros, Inc.
+ * 
+ * This file is part of Selenified.
+ * 
+ * Selenified is licensed under the Apache License, Version
+ * 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy 
+ * of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, 
+ * software distributed under the License is distributed on 
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY 
+ * KIND, either express or implied. See the License for the 
+ * specific language governing permissions and limitations 
+ * under the License.
+ */
+
+package com.coveros.selenified.selenium.element;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.log4testng.Logger;
+
+import com.coveros.selenified.exceptions.InvalidLocatorTypeException;
+import com.coveros.selenified.output.OutputFile;
+import com.coveros.selenified.selenium.element.Get;
+import com.coveros.selenified.selenium.element.Is;
+import com.coveros.selenified.selenium.element.WaitFor;
+import com.coveros.selenified.output.OutputFile.Result;
+import com.coveros.selenified.selenium.Selenium.Locator;
+import com.coveros.selenified.tools.General;
+
+public class Element {
```

Comment:
So this looks more like what I was expecting. The class is still a bit long but in this case it might not be worth it to refactor and shorten it. 

Once again, if you were to try and refactor it to make it a little more compact, I would consider a composition pattern instead of an extension pattern.  Putting the different types of behaviors, like click, type, move, etc into their own implementation classes and then pass them into the element. The element's click move and other methods would just call out to the action implementation class. That could allow you to test those mechanisms in isolation easily.  But it may also be overkill for what you need right now. Especially since I think Element is the only class that actually needs those behaviors. 

It would make more sense if there were different types of elements that each required some subset of the behaviors. You could just instantiate the concrete element with the proper subset of behaviors. Prob overkill in this case though